src/hooks/use-browser-resize.ts

```diff
@@ -0,0 +1,15 @@
+import { useEffect } from 'react';
+import { fromEvent, interval } from 'rxjs';
+import { throttle } from 'rxjs/operators';
+
+export const useBrowserResize = (onResize: (event: Event) => void): void => {

```

Comment:
Are you sure that takes a function and paste it in dependency is a good idea? I think that no everyone knows that it makes unnecessary re-renders